package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
 	"name": "kvue",
-	"version": "2.156.2",
+	"version": "2.157.0",
```

Comment:
Which `npm` version are you running? This change can be reverted.

Reply:
I am using version 8.5.0

Reply:
I think this is ok... it will get updated in the background again later...

Reply:
Ah this version will be auto-updated to match the package.json version.